docs/version0.18.md

```diff
@@ -27,42 +27,41 @@
 
 The following new features and notable changes since v 0.17.0 are included in this release:
 
-- [jextract now available on macOS for OpenJDK version 8](#jextract-now-available-on-macos-for-openjdk-version-8)
-- [New shared classes cache suboption to turn off timestamp checking](#new-shared-classes-cache-suboption-to-turn-off-timestamp-checking)
-- [`-Xmso` 1 MB minimum value on z/OS 64-bit](#-xmso-1-mb-minimum-value-on-zos-64-bit)
-- [New Java&trade; statistics monitoring (`jstat`) tool](#new-jstat-tool)
-- [-XX:+TransparentHugePage is enabled by default on more Linux systems](#-xxtransparenthugepage-is-enabled-by-default-on-more-linux-systems)
-- [Add new Xdump exit agent and ExitOnOutOfMemoryError option](#add-new-xdump-exit-agent-and-exitonoutofmemoryerror-option)
+- [`jextract` now available on macOS&reg; for OpenJDK version 8](#jextract-now-available-on-macos-for-openjdk-version-8)
+- [New shared-classes cache suboption to turn off timestamp checking](#new-shared-classes-cache-suboption-to-turn-off-timestamp-checking)
+- [`-Xmso` 1 MB minimum value on z/OS&reg; 64-bit](#-xmso-1-mb-minimum-value-on-zos-64-bit)
+- [`jstat`: new Java&trade; statistics monitoring tool](#jstat-new-java-statistics-monitoring-tool)
+- [`-XX:+TransparentHugePage` is enabled by default on more Linux&reg; systems](#-xxtransparenthugepage-is-enabled-by-default-on-more-linux-systems)
+- [New Xdump "exit" agent and `ExitOnOutOfMemoryError` option](#new-xdump-exit-agent-and-exitonoutofmemoryerror-option)
```

Comment:
We tend to refer to the agents as dump agents, not Xdump agents, so I think this item should be changed for consistency, including the section this links to. There is a confusing "See Xdump dump agents for more about agents." that would be better written something like "For more information about dump agents, see [`-Xdump`](https://eclipse.github.io/openj9-docs-staging/449/xdump/#dump-agents).